tasks/connectors/cylera/readme.md

```diff
@@ -12,15 +12,16 @@ To run this task, you require the following **Cylera** information:
 
 ## Incremental runs
 
-The Cylera task supports incremental runs. Use them to improve performance and pull only the latest changes rather than pulling all available assets and vulnerabilities every time. To benefit from incremental runs in Cylera, store the datetime when the connector is run outside toolkit and provide this value to toolkit for time-based filtering (such as, `cylera_last_seen_after`).
+The Cylera task supports incremental runs. Use them to improve performance and pull only the latest changes rather than pulling all available assets and vulnerabilities every time. Currently you cannot determine exactly which last_seen_after parameter was used for the previous run. So, the incremental run pulls data for one (1) additional day, expecting that pulling data from Cylera takes less then 1 day.
 
-The following steps illustrate using two variables to track when the last successful incremental run occurred and prevent missing data from a task failure.
+**Enabling Incremental runs**
+
+To enable incremental runs, use the following parameters:
+
+1. **incremental=true:** When enabled, the connector uses `last_connector_run_start_time`-`1 day` as value for `cylera_last_seen_after`
+
+2. **cylera_last_seen_after:** Use it only on the first run, while the following runs are using `last_connector_run_start_time`-`1 day`
```

Comment:
It looks good, the only thing that might be confusing here is that there's no need to remove `cylera_last_seen_after` for the 2nd and following runs. User can specify it each time, but it will only be used if there's no history of previous connector runs. When there's at least 1 successful connector run, `cylera_last_seen_after` is ignored.

Reply:
@jgarber-cisco yes, this is a good phrasing.

Reply:
Great, thanks. I can't modify @FPolillo's fork, so will have to get him to make the changes. Also asking Steve Albert to give Franco access to this repo so he doesn't have to fork.